src/20240409_AaveV2Ethereum_InterimAAMPLDistribution/InterimAAMPLDistribution.md

```diff
@@ -0,0 +1,56 @@
+
+This distribution have been defined the following way, with the help of @ChaosLabs and the Ampleforth team (for ubaAMPL holders):
+
+- From each address holding aAMPL, a percentage over the total aAMPL supply has been calculated, to understand how is the proportion of each address.
+- Using the previously calculated percentages, they have been applied over the total 300'000 USDC distribution: for example, for an address holding 5% of the total aAMPL, the claim has been calculated as 5% of 300'000; 15'000 USDC.
+- For the holders of aAMPL through [Unbuttoned aAMPL (ubaAMPL)](https://etherscan.io/token/0xf03387d8d0ff326ab586a58e0ab4121d106147df#balances), the proportion over the total supply of ubaAMPL has been used to calculate the claims on aAMPL. For example, if an address held 20% of the total ubaAMPL supply, and ubaAMPL itself would be 20% of the aAMPL supply, the claims of that address would be the 4% of the total aAMPL. We appreciate the Ampleforth team providing us these "internal" holdings of ubaAMPL, given their knowledge of the system.
+- Only claims over value of 30 USDC have been included, given that gas-wise, would not be profitable to claim lower amounts. However, those values lower than 30 USDC will be naturally taken into account for the final follow-up distribution.
+- Once again, _this is an interim distribution, that will be follow up by another with more a precise numbers and bigger in size_.
+- For the sake of reducing complexity, the Aave governance proposal will release the whole 300'000 USDC, and the Ampleforth team can just transfer to the Aave Collector the 40% of that amount, removing any dependency for users to claim as soon as possible.
+- It is possible to check each address claims [HERE](https://github.com/bgd-labs/aave-proposals-v3/blob/8d5b3e902adf7c5c246e752b5d6b6e0e5d9831b7/src/20240409_AaveV2Ethereum_InterimAAMPLDistribution/distribution.pdf).
+
+## Specification
+
+The distribution will be done via the [Merkl](https://app.merkl.xyz/) platform by Angle Labs, specialized on these operations and used before in other Aave DAO proposals, like the Merit program.
+
+Users with a balance below 30$ will be excluded from this initial distribution as the gas-cost for claiming would not offset set amount claimed.
+
+Therefore the proposal will perform the following steps upon execution:
+
+- withdraw USDC from the collector (298.5k including a 0.5% fee for angle labs)
+- approve the full amount to [0x8BB4C975Ff3c250e0ceEA271728547f3802B36Fd](https://etherscan.io/address/0x8BB4C975Ff3c250e0ceEA271728547f3802B36Fd) which is the distribution creator by Angle Labs
+- sign the tos of https://app.merkl.xyz/ via a onchain transaction, a requirement on the Merkl platform
+- create a campaign to distribute funds to the affected users
+
+2 hours after proposal execution, users will be able to claim the USDC on https://app.merkl.xyz/
+
```

Comment:
If for what i understood the campaign is only for 1 hour, it should be specified here

Reply:
Previous comment

Reply:
It's not the duration param is arbitrary but must be set due to the generic interface